plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -675,6 +675,22 @@ private static Map<String, String> getColumnNameToStatisticsName(JdbcTableHandle
         return columnNameToStatisticsName;
     }
 
+    @Override
+    protected String escapeObjectNameForMetadataQuery(String name, String escape)
+    {
+        requireNonNull(name, "name is null");
```

Comment:
Add a link to https://learn.microsoft.com/en-us/sql/t-sql/language-elements/like-transact-sql?redirectedfrom=MSDN&view=sql-server-ver16#arguments as a code comment.